packages/shared/src/viewer/helpers/state.ts

```diff
@@ -3,6 +3,19 @@ import type { MaybeNullOrUndefined, Nullable } from '../../core/helpers/utilityT
 import type { PartialDeep } from 'type-fest'
 import { UnformattableSerializedViewerStateError } from '../errors'
 
+declare enum MeasurementType {
```

Comment:
The `declare` is unnecessary, that keyword is usually only used in typing files when typing untyped JS code (.d.ts files)